2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Configure stats pid to 54284
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/.config/wandb/settings
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/settings
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_init.py:_log_setup():520] Logging user logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240520_121425-o2oexok7/logs/debug.log
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240520_121425-o2oexok7/logs/debug-internal.log
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_init.py:init():560] calling init triggers
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-20 12:14:25,237 INFO    MainThread:54284 [wandb_init.py:init():585] re-initializing run, found existing run on stack: tgy1pnei
2024-05-20 12:14:25,238 INFO    MainThread:54284 [wandb_run.py:_finish():2103] finishing run counterfactuals/sae_concept_eraser/tgy1pnei
2024-05-20 12:14:25,239 ERROR   MainThread:54284 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-05-20 12:14:25,239 INFO    MainThread:54284 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-20 12:14:25,239 INFO    MainThread:54284 [wandb_init.py:_jupyter_teardown():448] cleaning up jupyter logic
2024-05-20 12:14:25,239 INFO    MainThread:54284 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-20 12:14:25,239 INFO    MainThread:54284 [wandb_run.py:_restore():2326] restore
2024-05-20 12:14:25,239 INFO    MainThread:54284 [wandb_run.py:_restore():2332] restore done
